src/components/detailsSubpanel.jsx

```diff
@@ -3,13 +3,24 @@
 import React from 'react';
 
 class NodeDetailsSubpanel extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      expanded: this.props.expanded ? this.props.expanded : false
+    };
```

Comment:
We have now lost the ability to modify expanded state from outside the subpanel itself...